ci/check-params-env.sh

```diff
@@ -217,6 +247,22 @@ function check_image_variable_matches_name_and_commitref() {
         echo "Image URL points to an incorrect image: expected OPENSHIFT_BUILD_NAME '${expected_build_name}'; actual '${openshift_build_name}'"
         return 1
     }
+
+    # Check the size change constraints now
+    # 1. Percentual size change
+    percent_change=$((100 * actual_img_size / expected_img_size - 100))
```

Comment:
```suggestion
    percent_change=$((100 * actual_img_size / (expected_img_size - 100)))
```
is the 100 that is being substracted, is it SIZE_ABSOLUTE_TRESHOLD?

Reply:
No no; it's just a substraction so that the number is actually always between -100 and 100 interval. Let's count these examples:
1. actual image size is 150 ; expected image size is 100, then:
```
100 * 150 / 100 = 150% -> we need to substract 100 to have 50% change
```
2. actua limage size is 60 ; expected image size is 100, then:
```
100 * 60 / 100 = 60% -> we need to substract 100 to have -40% change (will be changed to absolute value 40% on the next line)
```

Maybe I can change the expression to this:
```
percent_change=$((100 * (actual_img_size / expected_img_size - 1) ))
```

The effect is still same. Let me know what you prefer and whether this all make sense to you :slightly_smiling_face:

Reply:
thank you for the explanation, it makes total sense now.
no change needed.